src/js/libcs/Accessors.js

```diff
@@ -219,6 +221,11 @@ Accessor.setField = function(geo, field, value) {
             geo.pinned = value.value;
         }
     }
+    if (field === "labeled" || field === "labelled") {
```

Comment:
Perhaps we should reconsider that approach one day, but that would be a separate PR.